src/components/Balance.jsx

```diff
@@ -1,9 +1,28 @@
+import styled from "styled-components";
 import { useSpin } from "../context/useSpin";
 import { formatCurrency } from "../utils/helpers";
 
+const StyledBalance = styled.div`
+  font-size: 20px;
+  font-weight: 500;
+  color: #363205;
+  margin-top: 5px;
+  margin-bottom: 10px;
+  display: block;
+  text-align: center;
+  width: 100%;
+  font-family: "Roboto", sans-serif;
+`;
+
 function Balance() {
   const { balance } = useSpin();
-  return <div>Balance: {formatCurrency(balance)}</div>;
+  return (
+    <StyledBalance>
+      {balance === 0
+        ? "Out of money! Please refresh the page to play again."
```

Comment:
I don't know how to go about in getting this to be randomly generated. Can you help @thuongho